src/components/Dialog/src/DialogLayer.vue

```diff
@@ -16,6 +16,7 @@
 					:class="$s.disableScroll"
 				/>
 				<v :nodes="dialogApi.state.vnode" />
+				<!-- <m-popover-layer /> -->
```

Comment:
can you please remove this line if it's not needed?

Reply:
Done!